lua/neotest-kotlin/init.lua

```diff
@@ -30,84 +30,100 @@ end
 ---@param root string Root directory of project
 ---@return boolean
 function M.Adapter.filter_dir(name, rel_path, root)
-  return filter.test_directory(name)
+	return filter.test_directory(name)
 end
 
 ---@async
 ---@param file_path string
 ---@return boolean
 function M.Adapter.is_test_file(file_path)
-  return filter.is_test_file(file_path)
+	return filter.is_test_file(file_path)
 end
 
 ---Given a file path, parse all the tests within it.
 ---@async
 ---@param file_path string Absolute file path
 ---@return neotest.Tree | nil
 function M.Adapter.discover_positions(file_path)
-  local positions = lib.treesitter.parse_positions(file_path, treesitter_query, {
-    nested_namespaces = true,
-    nested_tests = false,
-  })
+	local positions = lib.treesitter.parse_positions(file_path, treesitter_query, {
+		nested_namespaces = true,
+		nested_tests = false,
+	})
 
-  return positions
+	return positions
 end
 
+---Determines the package of a directory
+---@param dir string
+---@return string? package
+local function dir_determine_package(dir)
```

Comment:
Currently, this gets the directory and finds the first valid test file in there and just does

org.example.*

to run everything in that directory and its directories